### ISSUE TITLE
Combining this with HTTPS

### ISSUE THREAD

**Issue by DonaldTsang** [maxisoft/Freenom-dns-updater#21]
Is it possible to combine this application with https://letsencrypt.org/ or https://certbot.eff.org/ ?
that way things can get more streamlined.

**Comment by ghost** [comment]
Well i don't know if this is helpful or not but you can run certbot as always but with your freenom domain. Then you can enable systemctl for the provided job. 

The only issue i experienced is that DNS records are updated very slowly but it still works.

**Comment by DonaldTsang** [comment]
Yeah the idea is that both Freenom Updates and Certbot Updates can be automated together in one package, such that people can do minimum work.

**Comment by ghost** [comment]
I think that shouldn't be to difficult. Maybe one could write a shell script that periodically issues both updates. That would be a very hacky solution though.

**Comment by maxisoft** [comment]
I guess it's already possible with *certbot hooks* in dns mode as described here: https://certbot.eff.org/docs/using.html#hooks

**Comment by benthetechguy** [comment]
You're looking for [certbot-dns-freenom](https://github.com/Shm013/certbot-dns-freenom).
